code/game/objects/farming.dm

```diff
@@ -560,7 +560,7 @@
 
 /obj/structure/farming/plant/oat
 	name = "oat plant"
-	desc = "a oat plant."
+	desc = "A oat plant."
```

Comment:
An*

Reply:
Got it.

Reply:
Completed in https://github.com/Civ13/Civ13/pull/2903